OpenRA.Mods.Common/Traits/Player/EnemyWatcher.cs

```diff
@@ -85,11 +84,28 @@ public void Tick(Actor self)
 				if (lastKnownActorIds.Contains(actor.Actor.ActorID))
 					continue;
 
+				var notificationPlayed = playedNotifications.Contains(actor.Trait.Info.Notification);
```

Comment:
Not really. I want to keep the played voice notification per player.
`EnemyWatcher` is attached to the PlayerActor, but `AnnounceOnSeen` is per-actor.
This means that if I move it if 5 tanks come into view you you will get 5 notifications.